hw/ip/edn/data/edn.hjson

```diff
@@ -259,6 +259,52 @@
                 '''
         }
       ]
-    }
+    },
+    {
+      name: "ERR_CODE",
+      desc: "Hardware detection of error conditions status register",
+      swaccess: "rw",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "0",
+          name: "SFIFO_RESCMD_ERR",
+          desc: '''
+                This is set when a reseed command fifo error is set.
+                Writing a zero to this bit will reset it.
+                '''
+        }
+        { bits: "1",
+          name: "SFIFO_GENCMD_ERR",
+          desc: '''
+                This is set when a generate command fifo error is set.
+                Writing a zero to this bit will reset it.
+                '''
+        }
+        { bits: "28",
+          name: "FIFO_WRITE_ERR",
+          desc: '''
+                This is type of error when other source bits are set in this register.
+                This is the case where a full FIFO has been written.
```

Comment:
I don't think I understand this: what are "other source bits" here? Do you mean that the bit has meaning when at least one of `SFIFO_RESCMD_ERR` or `SFIFO_GENCMD_ERR` is true and, in that case, this bit shows that we have written to a full FIFO?

The same question applies for the next two bits too.

Reply:
Re-writing to remove any ambiguity.